braviarc/braviarc.py

```diff
@@ -231,6 +231,10 @@ def set_volume_level(self, volume):
     def turn_on(self):
         """Turn the media player on."""
         self._wakeonlan()
+        # Try using the power on command incase the WOL doesn't work
+        POWER_ON_CMD = 'AAAAAQAAAAEAAAAuAw=='
```

Comment:
Sorry couldn't get to this earlier but I have amended the code and used the get_power_command. I used the TvPower Command instead of the Mute command.